packages/fxa-auth-server/lib/senders/emails/templates/subscriptionAccountFinishSetup/index.mjml

```diff
@@ -0,0 +1,34 @@
+<%# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at http://mozilla.org/MPL/2.0/. %>
+
+<mj-section css-class="mb-6">
+  <mj-column>
+    <mj-image src="<%- icon %>" alt="<%- productName %> icon" title="<%- productName %>" css-class="product-icon"></mj-image>
```

Comment:
This might be the case across other subplat templates too that I haven't noticed but at least in this one, I think there's more space below the image in the old templates than in the new. Maybe we can add 4-8px of more space here?

Reply:
A partial for the icon will be created with updated spacing for margin-bottom.